src/app/ngrx/reducers/selectors/downloads-chart.selectors.spec.ts

```diff
@@ -78,6 +78,13 @@ describe('Downloads Chart Selectors', () => {
     it('should sort episode chart data by total biggest to smallest', () => {
       expect(getTotal(result[0].data)).toBeGreaterThanOrEqual(getTotal(result[1].data));
     });
+
+    it('should only show podcast data if not the same number of episode downloads datapoints', () => {
```

Comment:
Danika found a fun bug. Due to the way the reducers are just storing the array of data without regard for changing date ranges or other parameters, if you load the app with one date range, navigate around the pages, then load an extended date range, and navigate around the pages again, an error occurs in the loading stage when the podcast downloads array doesn't line up with the episodes.